### PR TITLE
Always sort content by popularity

### PR DESCRIPTION
**Pull request by phamann**
This refactors `create-email` function to always sort grouped content by popularity using pageview count from Redis.

- Create scheduled events to invoke `cache-content-views` every hour of day and model in CloudFormation
- Pass `contentSinceDays` as event source with above schedule
- Refactor `cache-content-views` to also set expiry on Redis keys
- Refactor `sort-content-by-most-viewed` to fetch all views in one request from Redis using `MGET`
- Tidy up `create-email/index` to always use sort function
- Fix tests

### CODE REVIEW

File: templates/stack.json [hunk 1]
```diff
@@ -31,6 +31,68 @@
...
+					"Ref" : "AWS::Region"
+				  },
+				  ":",
+				  {
+					"Ref" : "AWS::AccountId"
+				  },
+				  ":function:nextmyftemail_cache-content-views"
+				]
+			  ]
+			},
```
Comment (phamann) [inline templates/stack.json]: Really don't like that the formatting of this file has changed and more importantly that all of the ARNs have been put on multiple lines. 😠

Comment (laurieboyes) [inline templates/stack.json]: Yep sorry that's my overzealous editor formatting and nothing to do with the .editorconfig file, so we can totally revert it

File: functions/cache-content-views/index.js [hunk 2]
```diff
@@ -6,22 +6,28 @@ const getContentReadCounts = require('./lib/get-content-page-views');
 const metrics = require('./lib-shared/metrics');
 const logger = require('@financial-times/n-lambda').logger;
 
-const daysBack = 7;
+const EXPIRATION = 1000 * 60 * 60 * 24 * 7;
```
Comment (laurieboyes) [inline functions/cache-content-views/index.js]: need secs not millisecs

**Comment by laurieboyes** [comment]
👍  save for that one comment

**Pull request merged**